Dockerfile-api

```diff
@@ -1,7 +1,7 @@
 FROM       node:12-alpine
 
 # Workdir
-WORKDIR    contact-tracing
+WORKDIR    covid-green
```

Comment:
Uncertain but AFAIK are there cascading changes as a result of this?  WORKDIR sets the active directory, but AFAIK this may only impact the image itself?

Reply:
The only downstream issues would be things inside the container. Basically this just changes the folder the application code lives in and is run from. 

I made this change only as an attempt to keep names consistent.

Reply:
Technically, it should be noted that if anyone is running the image and mounting a file in (perhaps a `.env` file) then they would be impacted by this. But the standard way of passing env variables into a node process in a container is too set actual ENV variables, not pass a file in.